hypha/apply/projects/templates/application_projects/invoice_detail.html

```diff
@@ -16,12 +16,31 @@ <h5 class="heading heading--no-margin">{% trans "For" %}: {{ object.project.titl
 <div class="wrapper wrapper--sidebar wrapper--outer-space-medium">
     <div class="wrapper--sidebar--inner">
         <div class="card card--solid">
-            <p class="card__text"><b>{% trans "Status" %}:</b> {{ object.get_status_display }}</p>
+            <p class="card__text"><b>{% trans "Invoice Number" %}:</b> {{ object.vendor_document_number }}</p>
             {% is_vendor_setup request as show_vendor_information %}
             {% if show_vendor_information %}
             <p class="card__text"><b>{% trans "Vendor" %}:</b> {{ object.project.vendor.name }}</p>
             {% endif %}
-            <p class="card__text"><b>{% trans "Invoice Number" %}:</b> {{ object.vendor_document_number }}</p>
+            <p class="card__text"><b>{% trans "Lead" %}:</b> {{ object.project.lead }}</p>
+            <p class="card__text"><b>{% trans "Fund" %}:</b> {{ object.project.submission.page }}</p>
+        </div>
+        <div class="card card--solid flex" x-data="{ collapsed: true }">
```

Comment:
You are missing accessibility and html semantics for the clickable element. Take a look at dropdown web component.